src/fileio.cpp

```diff
@@ -874,13 +874,14 @@ void CSphWriter::SeekTo ( SphOffset_t iPos, bool bTruncate )
 //////////////////////////////////////////////////////////////////////////
 
 static int g_iIOpsDelay = 0;
-static int g_iMaxIOSize = 0;
+static const int g_iLimitIOSize = ( 1UL << 30 ); // same as write chunk limit
```

Comment:
It would be good to add a cite from `man 2 write` about why is the limit.

```
On Linux, write() (and similar system calls) will transfer at most 0x7ffff000 (2,147,479,552) bytes, returning the number of bytes actually transferred.  (This is true on both 32-bit and 64-bit systems.)
```
Also, according to this man, limit is more closer to 1<<31 (that is - (1<<31)-0x1000). But it seems not very important as hit the limit expected as very-very seldom

Reply:
will be done